admin/src/css/configuration.less

```diff
@@ -92,3 +92,80 @@ dl.horizontal {
     font-size: 100%;
   }
 }
+.message-queue {
+  .flex-row {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    cursor: default;
+
+    .col {
+      padding: 0 15px;
+      &.col-fixed {
+        flex: 0 0 160px;
+      }
+      &.col-grow {
+        flex: 1 0 400px;
+      }
+    }
+  }
+
+  .message-state {
+    display: flex;
+
+    &:before {
+      font-family: 'FontAwesome';
+      font-size: 16px;
+      margin-right: 10px;
+      display: block;
+    }
+    &.state-scheduled:before {
+      content: "\f073";
```

Comment:
Use constants for all these icons so they can be named usefully. `variables.less` already has some you could use.